packages/output-handler/src/index.ts

```diff
@@ -47,48 +138,20 @@ const processMessage = async (event: unknown) => {
 
 	for (const record of parsedEvent.data.Records) {
 		const transcriptionOutput = record.body.Message;
-
```

Comment:
the diff is hard to read. everything we were doing before in this block should be moved to `handleTranscriptionSuccess` with no other changes